pkg/eventprocessor/impl/executor/impl/txnscope_createtable_test.go

```diff
@@ -0,0 +1,67 @@
+package impl
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/go-tableland/internal/tableland"
+	"github.com/textileio/go-tableland/pkg/eventprocessor/eventfeed"
+	"github.com/textileio/go-tableland/pkg/eventprocessor/impl/executor"
+	"github.com/textileio/go-tableland/pkg/sqlstore/impl/system"
+	"github.com/textileio/go-tableland/pkg/tables/impl/ethereum"
+)
+
+func TestCreateTable(t *testing.T) {
+	t.Parallel()
+
+	t.Run("success", func(t *testing.T) {
+		t.Parallel()
+		ctx := context.Background()
+
+		ex, dbURI := newExecutor(t, 0)
+
+		bs, err := ex.NewBlockScope(ctx, 0)
+		require.NoError(t, err)
+
+		assertExecTxnWithCreateTable(t, bs, 100, "0xb451cee4A42A652Fe77d373BAe66D42fd6B8D8FF", "create table bar_1337 (zar text)") //nolint
```

Comment:
Another example of a helper `assertExecTxnWithXXXX(...)`. 
It makes it easy to execute an XXXX event in the pipeline, and assert all ran correctly. This allows for having shorter test logic in most places.